build/repo.targets

```diff
@@ -57,12 +62,32 @@
     </ItemGroup>
 
     <ItemGroup>
-      <FilesToExcludeFromSigning Include="%(NPMPackage.ArtifactPath)" />
+      <FilesToExcludeFromSigning Include="%(NPMPackage.ArtifactPath);" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="GetJavaArtifactInfo" Condition="'$(JAVA_HOME)' != ''">
```

Comment:
Looks like we should also check for `Exists('$(JAVA_HOME)/lib/tools.jar')` since `JAVA_HOME` can be set when a JRE with no build tools is installed.